File: Design_Pattern_and_Principles/BuilderPatternExample/BuilderTest.java
public class BuilderTest {
    public static void main(String[] args) {
        Computer basicPc = new Computer.Builder("Intel i3", "8GB").build();
        
        Computer gamingPc = new Computer.Builder("AMD Ryzen 9", "32GB")
                .setStorage("2TB NVMe SSD")
                .setGraphicsCard(true)
                .build();

        System.out.println(basicPc);
        System.out.println(gamingPc);
    }
}

File: Design_Pattern_and_Principles/BuilderPatternExample/BuilderTest.cs
using System;

public class BuilderTest {
    public static void Main(string[] args) {
        Computer basicPc = new Computer.Builder("Intel i3", "8GB").Build();
        
        Computer gamingPc = new Computer.Builder("AMD Ryzen 9", "32GB")
                .SetStorage("2TB NVMe SSD")
                .SetGraphicsCard(true)
                .Build();

        Console.WriteLine(basicPc);
        Console.WriteLine(gamingPc);
    }
}